### DISCUSSION TITLE
Why using Local::now() for migration generation?

### DISCUSSION THREAD

**Discussion by Animeshz** [SeaQL/sea-orm#935] [Q&A] [answered]
I believe it'd be better to use [`SystemTime`](https://stackoverflow.com/a/44378174/11377112), so that migrations can be created across the globe without restriction of localtime being less or more...

What do you think?

**Comment by billy1624** [comment, marked as answer]
Hey @Animeshz, welcome! Then, the migration file will be prefixed with unix timestamp? That's much less readable, I'd say.

**Reply by Animeshz** [reply]
No, don't need to change the format, just reduce the timezone offset from the date-time...

**Reply by billy1624** [reply]
But seems there is no easy way to format [`SystemTime`](https://stackoverflow.com/a/44378174/11377112) into `YYYYMMDD_HHMMSS` without the help of `time` or `chrono`

**Reply by Animeshz** [reply]
Hmm, true!

Then can we use [chrono::DateTime#with_timezone](https://docs.rs/chrono/0.4.10/chrono/struct.DateTime.html#method.with_timezone) to get the GMT+0:00 time?

**Reply by billy1624** [reply]
I think we can add a flag to generate migration prefix with GMT+00:00 time? So, it's opt-in

**Reply by billy1624** [reply]
Would you mind opening an issue (feature request) for this?
Thanks!! @Animeshz
